.travis.yml

```diff
@@ -1,9 +1,9 @@
 sudo: false
 language: elixir
 elixir:
-  - 1.2.3
+  - 1.5
 otp_release:
-  - 18.2.1
+  - 20.0
```

Comment:
I see from the changelog that you are requiring Elixir 1.5 or greater at this point. However Elixir 1.5 can run against a wider set of Erlang versions. How about using a matrix build here where we test against a somewhat wider range of Erlang versions? We can't expect everyone to have upgraded to the latest and greatest?